### PR TITLE
OCP/PCI-DSS: Add evaluation for the rest of the Req-6 section

### PR DESCRIPTION
**Pull request by JAORMX**
Most of these are applicable to the organization deploying OpenShift,
and not to the platform itself.

Signed-off-by: Juan Antonio Osorio Robles <jaosorior@redhat.com>

### CODE REVIEW

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now d884f0695eca4e70ac9f5f3eb5a50960bebb04f1

File: controls/pcidss_ocp4.yml [hunk 1]
```diff
@@ -1194,16 +1194,23 @@ controls:
     bespoke or custom software developed by a third party.
   levels:
   - base
-  notes: ''
-  status: pending
+  notes: |-
+    This controls is applicable to the payment entity running OpenShift
+    and not to the platform itself. However, the as part of the development
+    of the OpenShift Container Platform Red Hat does perform extensive
```
[the remark this answers is not in the document]
Comment (Vincent056) [reply controls/pcidss_ocp4.yml:1200]: s/controls/control

File: controls/pcidss_ocp4.yml [hunk 2]
```diff
@@ -1194,16 +1194,23 @@ controls:
     bespoke or custom software developed by a third party.
   levels:
   - base
-  notes: ''
-  status: pending
+  notes: |-
+    This controls is applicable to the payment entity running OpenShift
```
Comment (Vincent056) [inline controls/pcidss_ocp4.yml:1198]: s/controls/control

File: controls/pcidss_ocp4.yml [hunk 3]
```diff
@@ -1194,16 +1194,23 @@ controls:
...
   controls:
   - id: Req-6.3.1
     title: 6.3.1 Remove development, test and/or custom application accounts, user IDs,
       and passwords before applications become active or are released to customers.
     levels:
     - base
-    notes: ''
-    status: pending
+    notes: |-
+      This controls is applicable to the payment entity running OpenShift
```
Comment (Vincent056) [inline controls/pcidss_ocp4.yml:1210]: s/controls/control

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 0bde124c033ef86c7293ad655d754e895d9b9f5b

**Comment by JAORMX** [comment]
@sherine-k @Vincent056 thanks for the reviews!

**Review by Vincent056 — APPROVED** [review]
/lgtm

**Comment by JAORMX** [comment]
/retest

error provisioning cluster

**Comment by JAORMX** [comment]
/retest

```
level=error
level=error msg=Error: error updating LB Target Group (arn:aws:elasticloadbalancing:us-east-1:460538899914:targetgroup/ci-op-4nk05yxc-1708d-8bs7z-sint/9921bd8c052d3f0d) tags: error tagging resource (arn:aws:elasticloadbalancing:us-east-1:460538899914:targetgroup/ci-op-4nk05yxc-1708d-8bs7z-sint/9921bd8c052d3f0d): TargetGroupNotFound: Target groups 'arn:aws:elasticloadbalancing:us-east-1:460538899914:targetgroup/ci-op-4nk05yxc-1708d-8bs7z-sint/9921bd8c052d3f0d' not found
level=error msg=	status code: 400, request id: a2926afa-d052-4db1-af03-9a982632bf83
level=error
level=error msg=  on ../tmp/openshift-install-cluster-335236546/vpc/master-elb.tf line 99, in resource "aws_lb_target_group" "services":
level=error msg=  99: resource "aws_lb_target_group" "services" {
level=error
level=error
level=fatal msg=failed to fetch Cluster: failed to generate asset "Cluster": failed to create cluster: failed to apply Terraform: failed to complete the change 
```

ugh... seems the AWS account is having issues today

**Pull request merged**